.gitignore

```diff
@@ -4,3 +4,4 @@
 /.tox
 /dist
 .vscode/
+.idea
```

Comment:
please don't touch gitignore files in projects you don't own -- please rebase this out thanks!